[Update] Promscale link fixes (#1100)
timescale/docs: promscale/visualize-data/apm-experience.md

Before you begin importing APM dashboards:

* Add these data sources in Grafana:
    * [Jaeger data source][promscale-as-jaeger]
    * [PostgreSQL data source][promscale-as-postgresql]  
* Check that the `timescaledb_toolkit` extension is installed.
  To verify if the extension is installed, run this SQL query:
  `SELECT * FROM pg_extension WHERE extname='timescaledb_toolkit';` 

>   If the query returns no results, then the extension is not installed. For instructions to install the extension, see *Install the TimescaleDB toolkit extension* section.

Reviewer: ```suggestion
  If the query returns no results, then the extension is not installed. For 
  more information about installing the extension, see the 
  [toolkit extension installation documentation][install-extension].
```

@Rajakavitha1 Add the link at the bottom of the page, as well 👇
